inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -43,30 +43,6 @@
 NON_DIGIT = re.compile('[^\d]+')
 
 
-@hepnames.over('acquisition_source', '^541[10_].')
```

Comment:
Can you elaborate a bit? (maybe create an issue if not there yet, I tend to forget stuff :older_man: )